### PR TITLE
[#169167508] Added info screen for SPID and CIE

### PR DESCRIPTION
**Pull request by Alex193a**
Added an info screen for SPID and CIE.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#169167508](https://www.pivotaltracker.com/story/show/169167508): Come CIT, se non dispongo di SPID o CIE, voglio vedere uno screen che mostri info per acquisire questi strumenti.

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/screens/authentication/SpidInformationScreen.tsx [hunk 1]
```diff
@@ -37,83 +39,79 @@ const styles = StyleSheet.create({
...
+            <Text>{I18n.t("authentication.spid_or_cie")}</Text>
+          </View>
+          <Tabs
+            tabBarUnderlineStyle={styles.tabBarUnderline}
+            tabContainerStyle={styles.tabBarContainer}
+          >
+            <Tab heading={I18n.t("authentication.spid")}>
+              <Content>
+                <Markdown>
+                  {I18n.t("authentication.spid_information.spid")}
```
Comment (valentinacala) [inline ts/screens/authentication/SpidInformationScreen.tsx]: Can you center the number with respect to the item list height? 
Maybe it could be enought to insert `alignItems: 'center'` or `alignSelf: 'center'`
It is:
image

and should be:
image

If possible, try to increase the number font size.

Comment (Alex193a) [reply ts/screens/authentication/SpidInformationScreen.tsx]: As far as I know, the position of the number of an html list (<li>) cannot be changed, for this reason I used a smaller font. 
It might be possible to do with some other changes to CSS but I can't think of any simple method to do it.

Comment (valentinacala) [reply ts/screens/authentication/SpidInformationScreen.tsx]: To get it the list could be rendered by using again the `getValueContent` function

Comment (Alex193a) [reply ts/screens/authentication/SpidInformationScreen.tsx]: Getting back to using `getValueContent` would "nullify" the changes made by the PR https://github.com/teamdigitale/io-app/pull/1382 whose purpose is to be able to use lists with similar style as it is currently but with the `<Markdown>` component.
In any case, it's okay, I'll do it right now!

File: locales/en/cie.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+**EIC**, the Electronic Identity Card, allows you to access all the services of the Public Administration. You can use it to conveniently access IO on your smartphone. You will also need the pin that is given to you half in the registry and half by mail together with the card.
```
Comment (Undermaken) [inline locales/en/cie.md]:
```suggestion
**CIE**, the Electronic Identity Card, allows you to access all the services of the Public Administration. You can use it to conveniently access IO on your smartphone. You will also need the pin that is given to you half in the registry and half by mail together with the card.
```
Since it is always referred in italian we should keep the acronym. It could be confusing

File: locales/en/cie.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+# EIC is safe
+
+Together with SPID, EIC is the only tool that allows you to certify your digital identity with the highest level of security.
+
+# How to get EIC
+
+You can apply for the EIC at the registry office of your municipality of residence or residence: to learn more go to [www.cartaidentita.interno.gov.it](https://www.cartaidentita.interno.gov.it)
```
Comment (Undermaken) [inline locales/en/cie.md]:
```suggestion
You can apply for the EIC at the registry office of your municipality of residence: to learn more go to [www.cartaidentita.interno.gov.it](https://www.cartaidentita.interno.gov.it)
```

File: locales/en/index.yml [hunk 4]
```diff
@@ -213,6 +213,11 @@ authentication:
     card5-content: In this experimental version you can find in the upper-right corner
       the icons to report bugs or start a chat with the development team.
       We're waiting for your feedbacks!
+  cie_information_request: !include cie.md
+  spid_or_cie: To use IO you must have one of these digital tools, which certify your identity with certainty.
```
Comment (Undermaken) [inline locales/en/index.yml]:
```suggestion
  spid_or_cie: To use IO you must have one of these digital tools, which certify your identity.
```

File: locales/it/cie.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+**CIE**, la Carta di Identità Elettronica, ti permette di accedere a tutti i servizi della Pubblica Amministrazione. La puoi usare per accedere comodamente ad IO sul tuo smartphone. Ti servirà anche il pin che ti viene consegnto metà in anagrafe e metà per posta insieme alla carta.
```
Comment (Undermaken) [inline locales/it/cie.md]:
```suggestion
**CIE**, la Carta di Identità Elettronica, ti permette di accedere a tutti i servizi della Pubblica Amministrazione. La puoi usare per accedere comodamente ad IO sul tuo smartphone. Ti servirà anche il pin che ti viene consegnato metà in anagrafe e metà per posta insieme alla carta.
```

File: locales/it/cie.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+# CIE è sicura
+
+Insieme a SPID, CIE è l'unico strumento che consente di certificare la propria identità digitale con il massimo livello di sicurezza.
+
+# Come ottenere CIE
+
+Puoi fare richiesta delle CIE presso l'anagrafe del tuo comune di residenza o dimora: per saperne di più vai su [www.cartaidentita.interno.gov.it](https://www.cartaidentita.interno.gov.it)
```
Comment (Undermaken) [inline locales/it/cie.md]:
```suggestion
Puoi fare richiesta della CIE presso l'anagrafe del tuo comune di residenza o dimora: per saperne di più vai su [www.cartaidentita.interno.gov.it](https://www.cartaidentita.interno.gov.it)
```

File: ts/screens/authentication/SpidInformationScreen.tsx [hunk 7]
```diff
@@ -55,65 +68,89 @@ class SpidInformationScreen extends React.Component<Props, never> {
...
+                  primary={true}
+                  onPress={() => this.browseToLink("https://www.spid.gov.it")}
+                >
+                  <Text>
+                    {I18n.t("authentication.spid_information.knowMore")}
+                  </Text>
+                </Button>
+              </View>
+            </Tab>
+            <Tab heading={I18n.t("authentication.cie")}>
```
Comment (Undermaken) [inline ts/screens/authentication/SpidInformationScreen.tsx]: When selected the title goes in 2 rows (iPhone X)

File: .env.example [hunk 8]
```diff
@@ -8,7 +8,7 @@ MIXPANEL_TOKEN='b5c28d653684157f831d6b604d5e1956'
 ENABLE_TEST_IDP=YES
 GCM_SENDER_ID='317157111831'
 # If enabled the App allows authentication with CIE.
-CIE_AUTHENTICATION_ENABLED=NO
+CIE_AUTHENTICATION_ENABLED=YES
```
Comment (Undermaken) [inline .env.example]: default is NO, revert this change

File: .env.production [hunk 9]
```diff
@@ -8,7 +8,7 @@ PAGOPA_API_URL_PREFIX_TEST='https://acardste.vaservices.eu:443/pp-restapi-CD'
 MIXPANEL_TOKEN='b5c28d653684157f831d6b604d5e1956'
 ENABLE_TEST_IDP=YES
 GCM_SENDER_ID='317157111831'
-CIE_AUTHENTICATION_ENABLED=NO
+CIE_AUTHENTICATION_ENABLED=YES
```
Comment (Undermaken) [inline .env.production]: same of .example

File: ts/screens/authentication/SpidCIEInformationScreen.tsx [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import Markdown from "../../components/ui/Markdown";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+import variables from "../../theme/variables";
+
+type OwnProps = {
```
Comment (valentinacala) [inline ts/screens/authentication/SpidCIEInformationScreen.tsx]:
```suggestion
type Props = {
```

File: ts/screens/authentication/SpidCIEInformationScreen.tsx [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+    Linking.openURL(url).catch(() => {
+      Toast.show({ text: I18n.t("genericError") });
+    });
+  }
+
+  public render() {
+    return (
+      <BaseScreenComponent goBack={true}>
+        <Container>
+          <View style={{ margin: 24 }}>
```
Comment (valentinacala) [inline ts/screens/authentication/SpidCIEInformationScreen.tsx]: I suggest here to insert it into styles as `{padding: customVariables.contentPadding}`

File: ts/screens/authentication/SpidCIEInformationScreen.tsx [hunk 12]
```diff
@@ -0,0 +1,163 @@
...
+                  </Text>
+                </Button>
+              </View>
+            </Tab>
+            <Tab
+              heading={I18n.t("authentication.cie")}
+              activeTextStyle={{
+                fontSize: 14,
+                color: customVariables.brandPrimary
+              }}
```
Comment (valentinacala) [inline ts/screens/authentication/SpidCIEInformationScreen.tsx]: shift styles under the constant `styles`

File: locales/it/spid.md [hunk 13]
```diff
@@ -0,0 +1,5 @@
+**SPID**, il Sistema Pubblico di Identità Digitale, ti permette di accedere a tutti i servizi della Pubblica Amministrazione con un'unica Identità Digitale (username e password). [Per saperne di più](https://www.cartaidentita.interno.gov.it)
```
Comment (valentinacala) [inline locales/it/spid.md]:
```suggestion
**SPID**, il Sistema Pubblico di Identità Digitale, ti permette di accedere a tutti i servizi della Pubblica Amministrazione con un'unica Identità Digitale (username e password). [Per saperne di più](https://www.spid.gov.it)
```

File: locales/en/spid.md [hunk 14]
```diff
@@ -0,0 +1,5 @@
+**SPID**, the Public Digital Identity System, allows you to access all Public Administration services with a single Digital Identity (username and password). [Read more] (https://www.cartaidentita.interno.gov.it)
```
Comment (valentinacala) [inline locales/en/spid.md]:
```suggestion
**SPID**, the Public Digital Identity System, allows you to access all Public Administration services with a single Digital Identity (username and password). [Read more] (https://www.spid.gov.it)
```

File: locales/en/index.yml [hunk 15]
```diff
@@ -233,6 +233,11 @@ authentication:
     unknownCardTitle: Unknown card
     unknownCardContent: The card used does not seem to be an Electronic Identity Card (CIE).
     unknownCardHelpLink: What is the Electronic Identity Card?
+  cie_information_request: !include cie.md
+  spid_or_cie: To use IO you must have one of these digital tools, which certify your identity.
+  request_cie: Request EIC
```
Comment (valentinacala) [inline locales/en/index.yml]:
```suggestion
  request_cie: Request CIE
```

File: ts/screens/authentication/SpidCIEInformationScreen.tsx [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+                <View spacer={true} extralarge={true} />
+              </Content>
+              <View footer={true}>
+                <Button
+                  block={true}
+                  primary={true}
+                  onPress={() => this.browseToLink("https://www.spid.gov.it")}
+                >
+                  <Text>
+                    {I18n.t("authentication.spid_information.knowMore")}
```
Comment (Undermaken) [inline ts/screens/authentication/SpidCIEInformationScreen.tsx]: It should be "Ottieni SPID" https://projects.invisionapp.com/share/DSUE5POGWHA#/screens
